tools/pepsi/src/upload.rs

```diff
@@ -93,7 +127,16 @@ pub async fn upload(arguments: Arguments, repository: &Repository) -> anyhow::Re
         let hulk_directory = hulk_directory.clone();
         async move {
             let nao = Nao::new(nao_address.ip);
+
+            // check os-version
```

Comment:
Please don't repeat yourself and make your code [DRY](https://en.m.wikipedia.org/wiki/Don%27t_repeat_yourself). Remove this comment, we can see what you are doing. 😉

Reply:
Ok